join.spec.md

```diff
@@ -93,27 +93,27 @@ flowchart TB
     Clients-->Router
 ```
 
-<a name=def-producer>**Producers**</a> generate join schemas. This spec places requirements on join schema producers.
+<a name=def-producer>**Producers**</a> generate supergraphs. This spec places requirements on supergraph producers.
 
-<a name=def-consumer>**Consumers**</a> consume join schemas. This spec places requirements on join schema consumers.
+<a name=def-consumer>**Consumers**</a> consume supergraphs. This spec places requirements on supergraph consumers.
 
-<a name=def-composer>**Composers** (or **compilers**)</a> are producers which compose subgraph schemas into a join schema. This document places no particular requirements on the composition algorithm, except that it must produce valid join schemas.
+<a name=def-composer>**Composers** (or **compilers**)</a> are producers which compose subgraphs into a supergraph. This document places no particular requirements on the composition algorithm, except that it must produce a valid supergraph.
 
 <a namme=def-router>**Routers**</a> are consumers which serve a composed schema as a GraphQL endpoint. *This definition is non-normative.*
-  - Graph routers differ from standard GraphQL endpoints in that they are not expected to process data or communicate with (non-GraphQL) backend services on their own. Instead, graph routers receive GraphQL requests and service them by performing additional GraphQL requests. This spec provides guidance for implementing routers, but does not require particular implementations of query separation or dispatch, nor does it attempt to normatively separate routers from other join schema consumers.
-  - Routers will often join schema elements from the schema they present to clients via introspection ({join__Graph}, for example, will typically be omitted)
+  - Graph routers differ from standard GraphQL endpoints in that they are not expected to process data or communicate with (non-GraphQL) backend services on their own. Instead, graph routers receive GraphQL requests and service them by performing additional GraphQL requests. This spec provides guidance for implementing routers, but does not require particular implementations of query separation or dispatch, nor does it attempt to normatively separate routers from other supergraph consumers.
+  - Routers will often omit schema elements from the schema they present to clients via introspection ({join__Graph}, for example, will typically be omitted)
 
 <a name=def-endpoint>**Endpoints**</a> are running servers which can resolve GraphQL queries against a schema. In this version of the spec, endpoints must be URLs, typically http/https URLs.
 
-<a name=def-subgraph>**Subgraphs**</a> are the GraphQL schemas which were composed to form the join schema. They are declared as values on the special {join__Graph} enum.
+<a name=def-subgraph>**Subgraphs**</a> are GraphQL schemas which are composed to form a supergraph. Subgraph names and metadata are declared within the special {join__Graph} enum.
```

Comment:
I continue to find it strange that we are talking about these things as subgraphs but putting it on a join__Graph enum. But that's OK.

Reply:
Not fixed on my branch :)